sphinxext/opengraph.py

```diff
@@ -184,6 +184,15 @@ def get_tags(context: Dict[str, Any], doctree: nodes.document, config: Dict[str,
     if image_url:
         tags += make_tag("og:image", image_url)
 
+    # Add image alt text (either provided by config or from site_name)
+    ogp_image_alt = config["ogp_image_alt"]
+    if isinstance(ogp_image_alt, str):
+        tags += make_tag("og:image:alt", config["ogp_image_alt"])
+    elif config["ogp_image_alt"] and site_name:
+        tags += make_tag("og:image:alt", site_name)
+    elif config["ogp_image_alt"] and htp.text:
+        tags += make_tag("og:image:alt", htp.text)
```

Comment:
```suggestion
    ogp_image_alt = config["ogp_image_alt"]
    if isinstance(ogp_image_alt, str):
        tags += make_tag("og:image:alt", ogp_image_alt)
    elif ogp_image_alt and site_name:
        tags += make_tag("og:image:alt", site_name)
    elif ogp_image_alt and htp.text:
        tags += make_tag("og:image:alt", htp.text)
```

I think the `ogp_image_alt` variable should be reused instead of recalculated.

Reply:
Absolutely, thanks for catching that!